kong/clustering.lua

```diff
@@ -290,7 +296,10 @@ function _M.handle_cp_websocket()
   end)
 
   while not exiting() do
-    local ok, err = sem:wait(10)
+    local ok, err = sem:wait(1)
+    if exiting() then
```

Comment:
This should not be necessary because the `while` loop already checks for `exiting()`.

Reply:
I think it is always good idea to check `exiting` after a `wait` to just cancel early.